products/office.md

```diff
@@ -37,35 +38,35 @@ releases:
 
 -   releaseCycle: "2013"
     releaseLabel: "2013 SP1"
-    releaseDate: 2014-02-25
+    releaseDate: 2013-01-29
```

Comment:
Microsoft lists `2013-01-09` as start of cycle -> https://learn.microsoft.com/lifecycle/products/microsoft-office-2013
I would rather take this date because we are linking to this doc...
```suggestion
    releaseDate: 2013-01-09
```